contracts/SnappAuction.sol

```diff
@@ -252,35 +254,46 @@ contract SnappAuction is SnappBase {
     function applyAuction(
         uint slot,
         bytes32 _currStateRoot,
-        bytes32 _newStateRoot,
+        bytes32 newStateRoot,                 // Only needed the case of fallback
         bytes32 _orderHash,
```

Comment:
Do we still need the orderHash and standingOrder indices here assuming that it was checked in the bidding phase and now three minutes have passed (unlikely to see a reorg that deep)?

Reply:
I was considering removing the check from the bidding function since the applyAuction can be called without ever going through the bidding phase. Preference? If not, I would prefer to remove it completely.

Reply:
Hmh, not a strong feeling on my side. I feel like it's a good protection mechanism to have in case you submit a bid and afterwards the chain reorgs. On the other hand, reorgs that long should not happen. @josojo what do you think?

I think it's more important to have that check in the bidding phase (which comes immediately after the order collection period that could be affected by a reorg) than here, 3 minutes after a potential reorg could have an effect.

Reply:
Have we landed on any decisions about this? Opinions from @josojo are welcome.

If not I will put the hash checks into solution Bidding phase and leave them out of apply auction. This could be somewhat dangerous in the fallback case, but shouldn't matter all that much.

Reply:
Also, should I assume we are also going to move the `stateRoot` requirement check to be only in bidding phase as well?